src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Node/Edit.cshtml

```diff
@@ -23,7 +23,15 @@
     </div>
 
     <div class="mb-3">
-        <button class="btn btn-success save" type="submit">@T["Save"]</button>
+        <div class="btn-group mb-1">
+            <button class="btn btn-success save" type="submit" name="submit" value="save">@T["Save"]</button>
+            <button type="button" class="btn btn-success dropdown-toggle dropdown-toggle-split" data-reference="parent" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                <span class="visually-hidden">@T["Toggle Dropdown"]</span>
+            </button>
+            <div class="dropdown-menu">
+                <button class="dropdown-item save-continue" type="submit" name="submit" value="saveAndContinue">@T["and continue"]</button>
```

Comment:
So it's in line with the other such features. Also do the same for the content type editor.

```suggestion
                <button class="dropdown-item save-continue" type="submit" name="submit" value="SaveAndContinue">@T["and continue"]</button>
```